smdebug/core/hook.py

```diff
@@ -546,6 +541,8 @@ def set_mode(self, mode):
 
     def export_collections(self):
         num_workers = self._get_num_workers()
+        if num_workers == 1 and self.first_process is False:
```

Comment:
`self._get_num_workers()` returns one of the three values:
```
1. torch.distributed.get_world_size()
2. hvd.size()
3. 1
```
When is the value ≠ 1

Reply:
options 1 and 2 may return more than 1?

Reply:
Doesn't this then this mean that the distribution strategy is supported?

Reply:
self.first_process is set only when num_workers =1 , so just checking self.first_process = False should suffice. 
https://github.com/awslabs/sagemaker-debugger/commit/62a4e2ab8ea315e9ad575a06f6493bb9c2620b18